rest_api/templates/rest_framework/api.html

```diff
@@ -0,0 +1,51 @@
+{% extends "rest_framework/base.html" %}
+{% load static %}
+{% load rest_framework %}
+{% if is_pgs_app_on_gae == 0 %}
+  {% load compress %}
+{% endif %}
+
+{% block style %}
+    {% include "catalog/libs/css.html" %}
+
+    {% comment %} Used for the JSON code block {% endcomment %}
+    <link rel="stylesheet" type="text/css" href="{% static "rest_framework/css/prettify.css" %}"/>
+
+    {% if is_pgs_app_on_gae %}
+      <link rel="stylesheet" href="{% static 'rest_api/pgs_rest_web.min.css' %}" type="text/css" media="all" />
+    {% else %}
+        {% compress css %}
+            <link type="text/x-scss" href="{% static 'rest_api/pgs_rest_web.scss' %}" rel="stylesheet" media="screen"/>
+        {% endcompress %}
+    {% endif %}
```

Comment:
Won't be needed if we move the content of **pgs_rest_web.scss** into other existing scss files

Reply:
Replaced with pgs_rest.scss